test/local/routes/account_devices.js

```diff
@@ -0,0 +1,367 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The other two test files are called `..._routes.js`. Are we bothered by that inconsistency?

Actually, given that they're all in a directory called `routes` now anyway, can we remove the `_routes` suffix from the file name?

And then this one could just be `devices.js` without the `account_` prefix maybe?